### DISCUSSION TITLE
Does Trame has the capabilities to provide certain Paraview features?

### DISCUSSION THREAD

**Discussion by Kaoutar-87** [Kitware/trame#240] [Trame/VTK] [answered]
Hello,

I am reaching out to inquire about Trame's capabilities and whether or not it can onboard certain Paraview features. Specifically, I am hoping to select triangles and render them so that I can see the temperature on each triangle. Additionally, I would like to convert cell data to point data for a smoother, more continuous rendering, and I am also interested in creating clips to extract temperature data.

I am working with a .case file that exposes fluid domain results, and I am using Trame and vtk.js to visualize the data. To provide a clearer understanding of what I am trying to accomplish, I have included images that show the exact visualizations I am hoping to achieve using Trame.

Given this information, I am wondering if Trame has the capabilities to provide these features. If not, do you have any suggestions for alternative solutions or resources that may be helpful in achieving these visualizations?

Thank you for your time and assistance.

**Comment by jourdain** [comment, marked as answer]
All of it is possible in trame. Just need to be coded in Python on the server side. The trace in ParaView can help to some extent.
